pkg/reconciler/v1alpha1/autoscaling/hpa/hpa.go

```diff
@@ -135,8 +135,8 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 		// to status with this stale state.
 	} else if _, err := c.updateStatus(pa); err != nil {
 		logger.Warnw("Failed to update pa status", zap.Error(err))
-		c.Recorder.Eventf(pa, corev1.EventTypeWarning, "UpdateFailed",
-			"Failed to update status for PA %q: %v", pa.Name, err)
+		c.Recorder.Eventf(pa, corev1.EventTypeWarning, "InternalError",
```

Comment:
We use UpdateFailed everywhere else, I think.

Reply:
reverted the name, kept lowercase.